.ci/test_r_package_solaris.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sh build-cran-package.sh || exit -1
+
+apt-get install --no-install-recommends -y \
+  libcurl4-openssl-dev \
+  libxml2-dev \
+  libssl-dev
+
+log_file="$GITHUB_WORKSPACE/rhub_logs.txt"
```

Comment:
We try to not use CI service dependable env variables in bash script (try to search for `GITHUB_WORKSPACE`: it is used only in config `yml` files).

I guess we can switch to relative paths here just like in `test_r_package_valgrind.sh`
https://github.com/microsoft/LightGBM/blob/2f3764265cc2c43ffaa250e89c1f2efc859404ae/.ci/test_r_package_valgrind.sh#L5-L13

Reply:
good point! fixed in https://github.com/microsoft/LightGBM/pull/4503/commits/10deff608ebcd95b60abd310472bad181d9768e2